README.md

```diff
@@ -17,76 +16,119 @@ _A table of contents to help people find things_
 - [Licence](#licence)
 
 ## Usage
-_Whatever usage instructions your component has. We've broken this down by Markup, JavaScript and Sass, but it depends how complex your component is._
+`o-message` uses Sass and Javascript to show and hide a message component.  
+It can be initialised declaratively if markup is provided on the page, or it can be initialised imperatively when using the [manual build process](http://origami.ft.com/docs/developer-guide/modules/building-modules/).
+
+By default, `o-message` initialises an alert message, which provides information in response to a user action. It currently has three themes, 'success', 'neutral' and 'error', and relies on the markup (or configuration) to determine certain aspects of each theme's styles.
 
 ### Markup
 
-_Common templating can go here, especially if there is only one template, but people can always check the demos for more._
+This is an example of the declarative way of instantiating an error message that spans **across a viewport**.
+
+```html
+<div class="o-message o-message--alert o-message--alert-error" data-o-component="o-message">
+	<div class="o-message__container">
+		<div class="o-message__content">
+			<p class="o-message__content--highlight">Something went wrong!
+				<span class="o-message__content--detail">The quick brown fox did not jump over the lazy dogs.</span>
+			</p>
+			<div class="o-message__actions">
+				<a href="#" class="o-message__button o-message__action--primary">Button</a>
+				<a href="#" class="o-message__link o-message__action--secondary">Text link</a>
+			</div>
+		</div>
+		<a href="#void" class="o-message__close" role="button" aria-label='Close' title='Close'></a>
+	</div>
+</div>
+```
+_Note: at different viewport sizes, the message element hides the following elements:_
+- `<span class="o-message__content--detail">`
```

Comment:
I'm really surprised the detail element is hidden on smaller viewports. Seems like it will hold pretty important information related to the type of success/problem? Based on the readme `additional-info` feels less critical and  more reasonable to hide, but that isn't?

Reply:
So, based on James' design, (and I'm sure I could make this clearer in the readme so I shall attempt it) the regular alert message doesn't have the additional info, and it hides the detailed info at smaller viewport sizes:

<img width="834" alt="screen shot 2018-03-05 at 12 02 31" src="https://user-images.githubusercontent.com/16777943/36974043-267c0ec8-206d-11e8-9d68-18e1b5a7a294.png">

Where as the inline version has the additional info and doesn't react to viewport sizes, as it stretches (or shrinks) to whatever container it is put inside.

Reply:
👍 got it now after our chat! "inline" does not mean "display:inline", it refers to being in a page alongside content rather than spanning the top.